Remotion/ObjectBinding/Web.Development.WebTesting.IntegrationTests/ObjectBinding.Web.Development.WebTesting.IntegrationTests.csproj

```diff
@@ -178,15 +178,19 @@
     <EmbeddedResource Include="SavedTestScreenshots\BocListControlObjectTest\InternetExplorer.any.ScreenshotTest_TableContainer1.png" />
     <EmbeddedResource Include="SavedTestScreenshots\BocListControlObjectTest\InternetExplorer.any.ScreenshotTest_TableContainer2.png" />
     <EmbeddedResource Include="SavedTestScreenshots\BocTreeViewControlObjectTest\InternetExplorer.any.WebTreeView.png" />
-  </ItemGroup>
-  <ItemGroup>
     <EmbeddedResource Include="SavedTestScreenshots\BocListControlObjectTest\InternetExplorer.any.ScreenshotTest_TableContainer3.png" />
-  </ItemGroup>
-  <ItemGroup>
     <EmbeddedResource Include="SavedTestScreenshots\BocTreeViewControlObjectTest\Chrome.any.WebTreeView.png" />
-  </ItemGroup>
-  <ItemGroup>
     <EmbeddedResource Include="SavedTestScreenshots\BocAutoCompleteReferenceValueControlObjectTest\InternetExplorer.any.ScreenshotTest2.png" />
+    <EmbeddedResource Include="SavedTestScreenshots\BocAutoCompleteReferenceValueControlObjectTest\Firefox.any.ScreenshotTest.png" />

```

Comment:
quick demo of the firefox screenshots please. possibly compared to the chrome ones.